tests/testthat/helper.R

```diff
@@ -0,0 +1,8 @@
+library(mockery)
+library(tibble)
+
+cookies <- tibble(
```

Comment:
This is what you talked about yesterday!! Nice. tibble as a dependency good to remember.

Reply:
it's just for the tests, to be clear, the thing itself is still squeaky clean dependencywise